src/index.js

```diff
@@ -0,0 +1,17 @@
+import _ from 'lodash';
+import './style.css';
+import playersData from './modules/data.js';
+
+const scores = document.querySelector('#scores');
+
+playersData.forEach((data, i) => {
+  if (i % 2 === 1) {
+    scores.innerHTML += `
+        <div class="player-details even">${playersData[i].name} : ${playersData[i].score}</div>
+    `;
+  } else {
+    scores.innerHTML += `
+        <div class="player-details">${playersData[i].name} : ${playersData[i].score}</div>
+    `;
+  }
```

Comment:
[OPTIONAL] Well done here 💯 but you could consider using the ternary condition to simplify this, it would go like below 👇🏽 
```
scores.innerHTML += (i % 2 === 1) ? `<div class="player-details even">${playersData[i].name} : ${playersData[i].score}</div>` : `<div class="player-details">${playersData[i].name} : ${playersData[i].score}</div>`;
```